boards/arm/sam4s_xplained/doc/sam4s_xplained.rst

```diff
@@ -0,0 +1,159 @@
+.. _sam4s_xplained:
+
+SAM4S Xplained
+################
+
+Overview
+********
+
+The SAM4S Xplained evaluation kit is a development platform to evaluate the
+Atmel SAM4S series microcontrollers.
+
+Hardware
+********
+
+- ATSAM4S16C ARM Cortex-M4 Processor
+- 12 MHz crystal oscillator
+- internal 32.768 kHz crystal oscillator
+- IS66WV51216DALL 8 Mb SRAM
+- Micro-AB USB device
+- Micro-AB USB debug interface supporting SEGGER OB and Virtual COM Port and
+  Data
+- One reset and one user pushbutton
+- 2 yellow user LEDs
+- IC pads for external flash chip
+
+Supported Features
+==================
+
+The sam4s_xplained board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+| Ext SRAM  | on-chip    | external SRAM                       |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the Kconfig
+:file:`boards/arm/sam4s_xplained/sam4s_xplained_defconfig`.
+
+Connections and IOs
+===================
+
+The `SAM4S Xplained Online User Guide`_ has detailed information about board
+connections. You can find a PDF of the schematic at http://www.atmel.com/tools/sam4s-xpld.aspx?tab=documents.
```

Comment:
unless you REALLY want this url to be displayed in the doc, it would be better to give this URL a title and put the title here instead (as you did for the `SAM4S Xplained Online User Guide` above.